### DISCUSSION TITLE
Provide pointers to information about recovery of data from old backups

### DISCUSSION THREAD

**Discussion by masinter** [Interlisp/medley#2267] [General]
```
I don't remember doing anything with low-level networking in Medley or
predecessors at Savoir, but it's plausible John Sybalsky did, as
suggested by the filename.

None of the Savoir stuff is online in any place that I'm aware of.  I
have a box of backup tapes from that era, which might be interesting.  I
even found a matching QIC-150 tape drive in the garage, but I didn't
find the SCSI cables, terminators, interface card, etc. that would be
needed to connect it to a modern machine, and they don't appear to be
available.
```
_Originally posted by @tlipkis in https://github.com/Interlisp/medley/issues/786#issuecomment-1147102547_

**Comment by Michele31415** [comment]
On 1/13/2023 11:23 PM, Larry Masinter wrote:
>
> None of the Savoir stuff is online in any place that I'm aware of. I
> have a box of backup tapes from that era, which might be interesting. I
> even found a matching QIC-150 tape drive in the garage, but I didn't
> find the SCSI cables, terminators, interface card, etc. that would be
> needed to connect it to a modern machine, and they don't appear to be
> available.
>
I have a QIC-150 drive connected to my Sun Ultra 10.  Works fine. That 
said, it's hit or miss if tapes that old are still readable.

             - Michele

**Comment by pamoroso** [comment]
Can this issue be moved to Discussions?

**Comment by masinter** [comment]
Rather than moving to Discussions, i'd favor one of "do it" "leave it" or "close it".

* Do it: add a sentence somewhere that says something like: "magnetic tapes degrade over time; if you have a mag tape more than zNN (30?) years old, do _not_ try to read it without preparing the tape (or give a pointer to article on saving old recordings)". 
* leave it: just leave this issue here, open.
* close it: close the issue and forget about it.

**Comment by pamoroso** [comment]
In the do it case I'd go even further and advise anyone with a tape older than 10-20 years not to attempt recovery without expert help.

However, I recommend moving this to a discussion as it's not directly relevant as an issue but the information may still be useful to those googling. Or it's better to close it if not moved.

**Comment by pamoroso** [comment]
Any objections to moving this issue to a discussion?
